README.md

```diff
@@ -100,7 +102,25 @@ See [the elections documentation](elections) for further details.
 
 See the [vendoring documentation](VENDORING.md).
 
-# Vulnerability Disclosure
+# Vulnerability handling
+
+Handling vulnerabilities in Kata is handled by the Vulnerability Management Team (VMT).
```

Comment:
I'd drop the initial word to avoid the double usage of "handl*":

> Vulnerabilities in Kata Containers are handled by the Vulnerability Management Team (VMT).